### DISCUSSION TITLE
Showing Artists when using Android TV Client

### DISCUSSION THREAD

**Discussion by Wyvern2** [jellyfin/jellyfin-androidtv#2661] [General]
I am just getting familiar with Jellyfin. The server has been installed on a Raspberry Pi 4 with an external M2 SSD to hold the media files. A Samba share to my Windows 11 desktop computer allows me to easily transfer the files to the appropriate folders.

The Web Client on the desktop is used to configure the server. Everything works well. My main interest is transferring my Music CDs to  Jellyfin.

I particularly like the ability to add Artist (not just Album Artist) details, and now have an interesting range of artwork for the performers referred to on the individual CD tracks. 

Although the Artists can be displayed on the desktop computer, and the songs with which they are involved can be played, I cannot find this feature on the Android TV Client. My TV is linked to my HiFi system and is the final destination for the Music Files on the server. Has the Artist feature been implemented for the Android TV version? If so, how can it be enabled?

**Comment by nielsvanvelzen** [comment]
When you open a music library and go to the bottom row it should show an "Artists" button which you can use to browse by artist.

**Reply by Wyvern2** [reply]
Thank you for your reply. I have already tried this. The "Artists" button actually retrieves the "Album Artists" information.
The Desktop client is far more flexible and allows one to display "Album", "Album Artist", and "Artists" as desired.
It is the last category that I am hoping to have available.

**Reply by nielsvanvelzen** [reply]
Ok I see what you mean. It looks like adding both the artists and album artists buttons is quite easy (with the artists button showing the correct data). Should be available in the next update (0.16.0)!

**Reply by Wyvern2** [reply]
That's great news! I look forward to trying it in the next update. For example, I have a favourite musician who appears on occasional tracks in different Albums. He is not a very famous player, but I like his style. To be able to select him from the the Artists display makes the Jellyfin client very neat system.
